<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenAsDialog</class>
 <widget class="QDialog" name="OpenAsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open As</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="inputFileGroupBox">
     <layout class="QGridLayout" name="gridLayout_0">
      <item row="0" column="0">
       <widget class="QLabel" name="inputFileLabel">
        <property name="text">
         <string>Input File:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="inputFileEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="inputFileBrowseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
        <widget class="QLabel" name="tilesetLabel">
          <property name="text">
            <string>Tileset:</string>
          </property>
        </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="isTilesetWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_1">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="isTilesetYesRadioButton">
           <property name="text">
            <string>YES</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">isTilesetButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="isTilesetNoRadioButton">
           <property name="text">
            <string>NO</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">isTilesetButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="isTilesetAutoRadioButton">
           <property name="text">
            <string>Auto detect</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">isTilesetButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_1_1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="celSettingsGroupBox">
     <property name="title">
      <string>CEL/CL2 File Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_1">
      <item row="0" column="0">
       <widget class="QLabel" name="celClippedLabel">
        <property name="text">
         <string>Clipped:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="celClippedWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="celClippedYesRadioButton">
           <property name="text">
            <string>YES</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">celClippedButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="celClippedNoRadioButton">
           <property name="text">
            <string>NO</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">celClippedButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="celClippedAutoRadioButton">
           <property name="text">
            <string>Auto detect</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">celClippedButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2_0">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="celWidthLabel">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="celWidthEdit">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Frame width</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="tilSettingsGroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Tileset Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="minWidthLabel">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="minWidthEdit">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Subtile width</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="minHeightLabel">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="minHeightEdit">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Subtile height</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="tilFileLabel">
        <property name="text">
         <string>Til File:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="tilFileEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="tilFileBrowseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="minFileLabel">
        <property name="text">
         <string>Min File:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="minFileEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="minFileBrowseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="solFileLabel">
        <property name="text">
         <string>Sol File:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="solFileEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="solFileBrowseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="ampFileLabel">
        <property name="text">
         <string>Amp File:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="ampFileEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="ampFileBrowseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="openButtonsWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openCancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="isTilesetButtonGroup"/>
  <buttongroup name="celClippedButtonGroup"/>
 </buttongroups>
</ui>
